src/draco/io/mesh_io.cc

```diff
@@ -71,11 +72,19 @@ StatusOr<std::unique_ptr<Mesh>> ReadMeshFromFile(
     return std::move(mesh);
   }
   if (extension == "ply") {
-    // Wavefront PLY file format.
+    // Stanford PLY file format.
     PlyDecoder ply_decoder;
     DRACO_RETURN_IF_ERROR(ply_decoder.DecodeFromFile(file_name, mesh.get()));
     return std::move(mesh);
   }
+  if (extension == "stl") {
+    // STL file format.
+    StlDecoder stl_decoder;
+    StatusOr<std::unique_ptr<Mesh>> status_or_mesh =
+        stl_decoder.DecodeFromFile(file_name);
```

Comment:
you should be able to return directly from the stl_decoder:
  `return stl_decoder.DecodeFromFile(file_name);`
  
  Not sure why we don't do that with the other decoders (except gltf).

Reply:
Done